layouts/partials/coming-soon-header.html

```diff
@@ -0,0 +1,9 @@
+<div class="container website-coming-soon">
+  <h1 class="text-center white margin-bottom-0">[WEBSITE_NAME]<br/> Working Group</h1>
```

Comment:
Couldn't we set up some default params to give us a default look and feel, and allow for overrides? It would probably be easier to just configure it at the site level rather than having to replace this and lose upgrades in the future

Reply:
This file is meant to be overridden.

Reply:
The idea is that this section could be anything.

Reply:
This is also meant to be something that lives for about 2-3 months. I am not overly concerned about upgrades.

We could add params that users can define in the site config. If we think it's going to be useful. However, in my mind, this would be something that would vary from site to site.

Reply:
Fair enough. Just figured params would be easier then adding a file, but with it being short lived (something I hadn't thought of) the big argument of not recieving upgrades is moot. Good to go!